src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -105,6 +114,20 @@ public override string SubProtocol
                 websocketControl.SupportedProtocols.Add(subProtocol);
             }
 
+            if (MessageWebSocketClientCertificateSupported)
+            {
+                // Use the first app-provided client cert that can be successfully converted into a WinRT client cert (if any).
+                foreach (X509Certificate dotNetClientCert in options.ClientCertificates)
+                {
+                    RTCertificate winRtClientCert = ConvertDotNetClientCertToWinRtClientCert(dotNetClientCert);
+                    if (winRtClientCert != null)
```

Comment:
.NET Framework doesn't have an equivalent exception type/message. Are we OK with adding a UWP-specific exception here? I'm all for it, but just want to confirm that there's a precedent for that type of divergence in behavior.